code/game/objects/items/cigs_lighters.dm

```diff
@@ -920,6 +920,30 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 		user.update_inv_wear_mask()
 		user.update_inv_hands() //EndWS edit - Lightable e-cigarettes
```

Comment:
Could you remove these WS edit comments? Good practice to nuke them when you see them